### DISCUSSION TITLE
Docker resources availability

### DISCUSSION THREAD

**Discussion by pkonas** [ansys/pyansys-geometry#883] [Q&A] [answered]
Please, how can I access windows_binary.zip for successful build of docker container as an Ansys Channel Partner? Is there any chance to get it or to directly pull out the docker image for ACPs, or not? Examples are written for method  launch_modeler which assumes the local docker container. Does it mean that this "OpenSource" fork is available only for Ansys employees? Thx, Petr.

**Comment by RobPasMue** [comment]
Hi @pkonas!

Sorry I missed this discussion. The ``zip`` files you mention are only available for Ansys employees and customers on demand (after it is requested through the dedicated channels).

We are planning on adding images to the Ansys Customer Portal (or at least the means to build them) in the medium term so that customers can easily access resources from it. Another planned activity is to provide instructions on how to build the Geometry Service Docker image from an Ansys installation in your local machine - I will open an issue for this last point right away.

Sorry for any inconveniences.

(pinging @ansys/pyansys-geometry-maintainers for visibility)

**Reply by RobPasMue** [reply, marked as answer]
Hi @pkonas - we just provided a script so that you can build your own Geometry Service Docker Image based on your Ansys Installation. Please give it a try and let me know your feedback. Instructions can be found here:

https://geometry.docs.pyansys.com/version/stable/getting_started/docker/windows_container.html#build-from-available-ansys-installation

Make sure to:
* Have the Geometry Service installed as part of your Ansys installation. You can force its installation by running ``setup.exe -silent -geometryservice`` where ``setup.exe`` is the Ansys installer
* Have Docker installed on your machine
* Make sure that you are running the Docker Windows engine (see https://geometry.docs.pyansys.com/version/stable/getting_started/docker/windows_container.html#docker-for-windows-containers for related instructions).

Let me know what's your feedback. I'll mark this response as the answer to the discussion.

**Comment by pkonas** [comment]
Hi Robero,
 thank you for your response. I forwarded your info to my team and we will
inform you asap.
BR, Petr.

st 24. 1. 2024 v 9:19 odesílatel Roberto Pastor Muela <
***@***.***> napsal:
